nixos/modules/services/web-apps/stash.nix

```diff
@@ -0,0 +1,95 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.stash;
+in
+{
+  options = {
+    services.stash = {
+      enable = mkEnableOption "An organizer for your porn, written in Go.";
+
+      package = mkPackageOption pkgs "stash" { };
+
+      dataDir = mkOption {
+        type = types.str;
+        default = "/var/lib/stash";
+        description = "The state directory where stash stores its metadata, cache and generated files.";
+      };
+
+      user = mkOption {
+        type = types.str;
+        default = "stash";
+        description = "User account under which stash runs.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "stash";
+        description = "Group under which stash runs.";
+      };
+
+      ip = mkOption {
+        type = types.str;
+        default = "0.0.0.0";
+        example = "::1";
+        description = "The ip that the server should bind to.";
+      };
+
+      port = mkOption {
+        type = types.port;
+        default = 9999;
+        description = "The port that the server should listen on.";
+      };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = lib.mdDoc "Open ports in the firewall for the stash web interface";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.services.stash = {
+      description = "Stash";
```

Comment:
This says nothing more than the name of the service itself. Please either provide a more descriptive description, or remove it altogether.